misk/web/tabs/web-actions/src/ducks/webActions.ts

```diff
@@ -154,7 +154,7 @@ export const WebActionInternalLabel: { [key: string]: string } = {
   "Allowed Capabilities": "allowedCapabilities",
   "Allowed Services": "allowedServices",
   "Application Interceptor": "applicationInterceptors",
-  "Dispatch Mechanism": "dispatchMechanism",
+  "Dispatch Mechanism": "httpMethod",
```

Comment:
Can we update this to "HTTP Method" as well?

Reply:
YES!

Reply:
Fixed.